helix-core/src/main/java/org/apache/helix/task/AbstractTaskDispatcher.java

```diff
@@ -74,6 +75,12 @@ public void updatePreviousAssignedTasksStatus(
       Set<Integer> skippedPartitions, WorkflowControllerDataProvider cache,
       Map<String, Set<Integer>> tasksToDrop) {
 
+    // If a job is in the following states and contains running tasks, all of the running tasks
+    // should go to the ABORTED states.
+    Set<TaskState> jobStatesForRunningTaskToAbortedState =
```

Comment:
I think calling it `jobStatesForAbortingTasks` does the trick and is shorter. What do you think?

Reply:
Yeah I wasn't happy with previous name as well. Change it as you suggested. Thanks.